app/Commands/LaunchCommand.php

```diff
@@ -167,14 +174,32 @@ private function detectPhpVersion()
         return $resultVersion;
     }
 
-    private function generateFlyToml(string $appName, string $nodeVersion, string $phpVersion, TomlGenerator $generator)
+    private function setProcesses( array $processes=[] ): array
+    {
+        $cronCommand = "cron -f";
+        if( $this->confirm("Do you want to run Laravel's scheduler via cron? This will run \"$cronCommand\" in a separate process group." ) )
```

Comment:
I'd use `$this->choice` with multi-select enabled here, so it would display a menu like this: 
What extra processes do you want to run?
[ ] Scheduler - This will run $cronCommand in a separate process group
[ ] Queue workers - This will run $workerCommand in a separate process group

Something along those lines would enable the user to set up 2 process groups with one interaction.